mirai_core/updater.py

```diff
@@ -105,7 +105,7 @@ async def handshake(self):
                 await self.bot.handshake()
                 if self.use_websocket:
                     asyncio.run_coroutine_threadsafe(
-                        self.bot.create_websocket(self.event_caller, self.handshake), self.loop)
```

Comment:
Why `self.handshakes` is removed?

Reply:
Sorrr, I didn't see the `return True` line below. Please check the new commit.
I'm not pretty sure which way is better, utilizing `ws_on_connection_close` or relying on `while True`?

Reply:
The idea is to retry indefinitely until connecting is established. So two scenarios:
1. Connection aborted due to any reason
2. Connection never established due to server down
This is the initial idea behind while True. If you have better idea, feel free to replace it.